apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/ValueConversion.cs

```diff
@@ -44,6 +44,11 @@ public static object ConvertToClrType(this Value wireValue, V1.Type spannerType,
                 }
                 return null;
             }
+            if (targetClrType == typeof(object))
```

Comment:
I *sort of* agree, but it feels really odd to have this as a special-case. (What about `typeof(ValueType)`? What about `IComparable`? There are all kinds of other types that could fit multiple options...) Let's leave it like this for now, but I'll have a look and think further.